src/main.js

```diff
@@ -1,23 +1,43 @@
 import domains from './config/domains';
 import createDomainWithActions from './utils/createDomainWithActions';
 
+const mergeDomains = (originalActions, additionalActions) =>
+  Object.keys(additionalActions).reduce((acc, val) => {
```

Comment:
PP: I'd find it a bit easier to read if `acc` and `val` were named more specifically. E.g. `domains` or `mergedDomains` for acc and `domainKey` for val might work.

Reply:
Yeah you're right, I was planning on renaming them (@Kemosabert is my witness) but I just forgot.

I'll change it.